sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveUDFSuite.scala

```diff
@@ -141,13 +141,11 @@ class HiveUDFSuite extends QueryTest with TestHiveSingleton with SQLTestUtils {
   }
 
   test("SPARK-2693 udaf aggregates test") {
-    checkAnswer(
-      sql("SELECT percentile(key, 1) FROM src LIMIT 1"),
-      sql("SELECT double(max(key)) FROM src"))
+    checkAnswer(sql("SELECT percentile(key, 1) FROM src LIMIT 1"),
+      sql("SELECT max(key) FROM src").collect().toSeq)
```

Comment:
It seems to be a big behavior change in Spark testing.
So, this PR is going to enforce us to use explicit `collect().toSeq` for `checkAnswer` in some cases?